README.md

```diff
@@ -1,68 +1,87 @@
 [![Coverage Status](https://coveralls.io/repos/github/Adyen/adyen-python-api-library/badge.svg?branch=master)](https://coveralls.io/github/Adyen/adyen-python-api-library?branch=master)
 
-# Adyen APIs Library for Python
-
-This library simplifies working with Adyen APIs and allows you to integrate Adyen
-payments within any Python application.
-
-## Integration
-The Library supports all APIs under the following services:
-
-* checkout
-* checkout utility
-* payments
-* modifications
-* payouts
-* recurring
-
-## Requirements
-
+This is the officially supported python library for using Adyen's APIs.
+## Integration 
+The library supports all APIs under the following services:
+ 
+* [Checkout API](https://docs.adyen.com/api-explorer/#/CheckoutService/v67/overview): Our latest integration for accepting online payments. Current supported version:  **v67**
+* [Payments API](https://docs.adyen.com/api-explorer/#/Payment/v64/overview): Our classic integration for online payments. Current supported version:  **v64**
+* [Recurring API](https://docs.adyen.com/api-explorer/#/Recurring/v49/overview): Endpoints for managing saved payment details. Current supported version:  **v49**
+* [Payouts API](https://docs.adyen.com/api-explorer/#/Payout/v64/overview): Endpoints for sending funds to your customers. Current supported version:  **v64**
+* [Orders API](https://docs.adyen.com/api-explorer/#/CheckoutService/v67/post/orders): Endpoints for creating and canceling orders. Current supported version:  **v67**
+* [Utility API](https://docs.adyen.com/api-explorer/#/CheckoutService/v1/post/originKeys): This operation takes the origin domains and returns a JSON object containing the corresponding origin keys for the domains. Current supported version:  **v1**
+ 
+For more information, refer to our [documentation](https://docs.adyen.com/) or the [API Explorer](https://docs.adyen.com/api-explorer/).
+ 
+ 
+## Prerequisites
+ 
+-   [Adyen test account](https://docs.adyen.com/get-started-with-adyen)
+-   [API key](https://docs.adyen.com/development-resources/api-credentials#generate-api-key). For testing, your API credential needs to have the [API PCI Payments role](https://docs.adyen.com/development-resources/api-credentials#roles).
 - Python 2.7 or 3.6
 - Packages: requests or pycurl ( optional )
-- Adyen account. If you don't have this you can request it here: https://www.adyen.com/home/discover/test-account-signup#form
+ 
 
-## Installation
+ ## Installation
 
 ### For development propose
 
-Clone this repository and run ```make install```
+Clone this repository and run 
+~~~~ bash 
+make install
+~~~~
 
 ### For usage propose
 
-Use pip command: ```pip install Adyen```
-
-## Usage
-
-Create a class instance of the 'Adyen' class.
-
-```python
+Use pip command: 
+~~~~ bash 
+pip install Adyen
+~~~~
+
+## Using the library
+ 
+ 
+### General use with basic auth
```

Comment:
should we show here the use with the API key as a default?

Reply:
I used the existed example, I can add the example from e2e